pkg/cmd/server/start/master_args.go

```diff
@@ -182,6 +185,19 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 
 	dnsServingInfo := servingInfoForAddr(&dnsBindAddr)
 
+	overrideConfig := configapi.ClusterResourceOverrideConfig{}
```

Comment:
The config file has to be made available to go into the web console asset bundle for the console to display appropriate creation options; this was part of the feature specification. That bundle is built from the AssetConfig in the MasterConfig, and this is where that's defined. I don't think there's any other reason for the AssetConfig to even be in there; I would agree there should probably be some other way to transmit this info but at the moment there isn't. I wasn't going for rewriting anything I don't have to.

the admission controller itself has no actual access to the master config when initialized; it has only the admission plugin config and a filename. there is no way to insert things into the asset configurati